views/base.html

```diff
@@ -55,6 +55,9 @@
     </div> -->
 
     <main>
+      {% block communities %}
```

Comment:
I'm still doing something wrong -- still fail on the includes. Do I need to tell node that the new include exists somehow? It "can't find the template."